src/app/home/home.component.ts

```diff
@@ -21,5 +24,10 @@ export class HomeComponent implements OnInit {
       false,
       'https://images-production.thebiggive.org.uk/0011r00002IMRknAAH/CCampaign%20Banner/db3faeb1-d20d-4747-bb80-1ae9286336a3.jpg',
     );
+
+    const openingDate = new Date("2023-01-09T12:00:00.000+00:00");
```

Comment:
I would have rather called the timezone Europe/London instead of +00:00, but I don't know if that's supported natively by  browsers.

Reply:
Not essential now, but if we don't have an easy way to this it might be worth bringing in a library like Luxon at some point ( https://moment.github.io/luxon/index.html#/zones)  - although that still doesn't solve the problem of relying on the browser's clock.

Reply:
As it's such a short-term change I think this is fine for now.

Reply:
@bdsl Angular already has powerful pipes for this stuff – I doubt we need more libraries